src/Logs/HttpAccessLog.php

```diff
@@ -47,6 +45,8 @@ protected function fillMatches(array $matches = []): void
 
     public static function parseDatetime(?string $datetime): ?CarbonInterface
     {
-        return $datetime ? Carbon::createFromFormat('d/M/Y:H:i:s O', $datetime) : null;
+        $timezone = config('log-viewer.timezone', config('app.timezone', 'UTC'));
+
+        return $datetime ? Carbon::createFromFormat('d/M/Y:H:i:s O', $datetime, $timezone) : null;
```

Comment:
this changes the intended behaviour of the `log-viewer.timezone` configuration value.

Passing the third parameter to `Carbon::createFromFormat` means the given time is in that specified timezone. Logs are always logged with the application's timezone (`config('app.timezone')`), so they should always be parsed with that timezone as well.

The Log Viewer's time zone parameter defines what timezone the date should be displayed as in the frontend, not what it should be parsed as.

For example, the application itself might have the time zone of UTC, because maybe the server itself is in London. But the developer is in Berlin and would like to view Log Viewer logs in their local timezone, so that's when they would update the `log-viewer.timezone` to the `Europe/Berlin` timezone so that all log times would get converted to their local time.

Reply:
Well noted, this changes the way the datetime is analyzed. I removed the timezone from the function and added a ->setTimezone after the function, similar to tz (previous)